subworkflows/local/bam_variant_calling_somatic_all/main.nf

```diff
@@ -178,26 +178,23 @@ workflow BAM_VARIANT_CALLING_SOMATIC_ALL {
         out_msisensorpro = out_msisensorpro.mix(MSISENSORPRO_MSI_SOMATIC.out.output_report)
     }
 
-    // MUTECT2
-    if (tools.split(',').contains('mutect2')) {
```

Comment:
I'd like to keep that very much please

Reply:
Of course we need it, I put it back now.